app/services/edition_unwithdrawer.rb

```diff
@@ -33,9 +33,8 @@ def fire_transition!
     unwithdrawn_edition.minor_change = true
     unwithdrawn_edition.editorial_remarks << EditorialRemark.create(author: user, edition: edition, body: "Unwithdrawn")
 
-    Edition::AuditTrail.acting_as(user) do
-      Whitehall.edition_services.force_publisher(unwithdrawn_edition).perform!
-    end
+    EditionForcePublisher.new(unwithdrawn_edition).send(:prepare_edition)
```

Comment:
Could we get around this by subclassing `EditionForcePublisher` instead of `EditionPublisher`? Either way, now you've got it working we should try and refactor this out as having to do this is a smell. 💩